estate_account/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, Command
+
+
+class EstateProperty(models.Model):
+    # ---------------------------------------- Private Attributes ---------------------------------
+
+    _inherit = "estate.property"
+
+    # ---------------------------------------- Action Methods -------------------------------------
+    def action_sold(self):
+        res = super().action_sold()
+        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
```

Comment:
```suggestion
        journal = self.env.ref("my_account_journal")
```

I think it should be better to create an Account Journal from data demo and then search it up here